archey/entries/window_manager.py

```diff
@@ -64,6 +73,18 @@ def __init__(self, *args, **kwargs):
                 r"(?<=Name: ).*",
                 check_output(["wmctrl", "-m"], stderr=DEVNULL, universal_newlines=True),
             ).group(0)
+
+            # Check Display-Server-Protokoll
+
+            session = os.environ.get('XDG_SESSION_TYPE', '')
+            if session == "x11":
+                session = "X11"
+            elif session == "wayland":
+                session = "Wayland"
+
+            if session != "":
+                self.value = self.value + " (" + session + ")"
+
```

Comment:
I guess this feature (?) is a separate concern that maybe should be proposed/discussed in another PR ?